.werft/observability/manifests/monitoring-satellite.yaml

```diff
@@ -44,5 +44,7 @@ imports:
       path: operations/observability/mixins/meta/rules
     - gitURL: https://github.com/gitpod-io/gitpod
       path: operations/observability/mixins/IDE/rules
+    - gitURL: https://github.com/gitpod-io/gitpod
+      path: operations/observability/mixins/self-hosted/rules/cardinality-analysis
```

Comment:
Oh, actually this can't be tested in the preview environment because the file is not on `main` yet 😅 

Unfortunately, there is no way of testing this one besides merging and checking results